charts/csi-secret-provider-class/templates/secretproviderclass.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+{{ template "common.metadata" $ }}
+spec:
+  provider: {{ required "Specify a valid provider." .Values.provider }}
```

Comment:
if default value for `provider` is `~` why set it as `required` ?

Reply:
the falsy value of `~` triggers the required warning.